Description: Use systrem zlib
 Don't download a random zlib, dynamically link against the OS provided version 
Index: plink-ng/Makefile
===================================================================
--- plink-ng.orig/Makefile	2016-06-06 13:26:17.224539505 +0100
+++ plink-ng/Makefile	2016-06-06 13:29:49.229289829 +0100
@@ -1,10 +1,10 @@
 # This is a bit of a mess.  Work with Makefile.std instead.
 
-CFLAGS=-Wall -O2
+CFLAGS=-Wall -O2 -DDYNAMIC_ZLIB
 BLASFLAGS=-L/usr/lib64/atlas -llapack -lcblas -latlas
 BLASFLAGS64=-L/usr/lib64/atlas -llapack -lcblas -latlas
 LINKFLAGS=-lm -lpthread
-ZLIB=zlib-1.2.8/libz.so.1.2.8
+ZLIB=-lz
 ARCH64=-arch x86_64
 
 UNAME := $(shell uname)
Index: plink-ng/Makefile.std
===================================================================
--- plink-ng.orig/Makefile.std	2016-06-06 13:26:17.224539505 +0100
+++ plink-ng/Makefile.std	2016-06-06 13:26:17.220539491 +0100
@@ -22,7 +22,7 @@
 CFLAGS=-Wall -O2
 BLASFLAGS=-L/usr/lib64/atlas -llapack -lcblas -latlas
 LINKFLAGS=-lm -lpthread -ldl
-ZLIB=zlib-1.2.8/libz.so.1.2.8
+ZLIB=-lz
 
 ifeq ($(SYS), MAC)
   GCC_GTEQ_43 := $(shell expr `g++ -dumpversion | sed -e 's/\.\([0-9][0-9]\)/\1/g' -e 's/\.\([0-9]\)/0\1/g' -e 's/^[0-9]\{3,4\}$$/&00/'` \>= 40300)
Index: plink-ng/dose2plink.c
===================================================================
--- plink-ng.orig/dose2plink.c	2016-06-06 13:26:17.224539505 +0100
+++ plink-ng/dose2plink.c	2016-06-06 13:26:17.220539491 +0100
@@ -96,7 +96,7 @@
   #endif
 #endif
 
-#include "zlib-1.2.8/zlib.h"
+#include <zlib.h>
 
 #ifdef __APPLE__
   #include <sys/sysctl.h>
